contracts/contracts/vault/VaultCore.sol

```diff
@@ -92,7 +90,7 @@ contract VaultCore is VaultStorage {
         emit Mint(msg.sender, priceAdjustedDeposit);
 
         // Rebase must happen before any transfers occur.
-        if (unitAdjustedDeposit >= rebaseThreshold && !rebasePaused) {
```

Comment:
These thresholds are going to be different for OETH.

Reply:
Yup I've added a TODO some time ago to double check these values here: 
https://github.com/OriginProtocol/origin-dollar/pull/1271/files#diff-a164c283475bded9c293a6f65d3256cf31a805c1fbf900425336b5095b2a08e0R118-R127